PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,60 @@ This governance policy sets forth the proposal process for projects to be accept
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
+
+##### Application Manager
+
+The application manager ensures progress through the process, helps the project understand the work to be completed and
+to move though the application process. Typically the application manager will be one of the Foundation staff members.
+
+##### Application Champion
+
+The application champion is a member of the CPC who commits to do the following:
+
+1. Provide weekly update to the applicant on progression through the process.
+1. Be available to answer questions from the applicant.
+1. Advocate on behalf of the project applying.
+1. Help to explain why the project is a good fit.
+1. Help the application manager when needed.
+
+
+#### Process
 
 1. Initial email sent to new-projects@lists.openjsf.org with filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md).
+1. An acknowledgement is sent to the applicant by Foundation staff within 48 hours (striving to reply within 24 hours when possible).
 1. Silent period. Internal only discussion.
-1. Initial acceptance as [Incubating][] Project.
+1. Initial acceptance as [Incubating][] Project. Before public announcement there will be:
+   * Private email to the CPC members which asks for +1 or objections to the application becoming public. This email will include:
+     * The name of the application manager for the application
+     * The name of the application champion for the application
+     * Completed application 
+     * Completed application-fit checklist.
+     * Confirmation that the board has had an FYI and there were no objections.
+     * Confirmation that there are no existing objections from CPC members.
+     This is not a formal vote just the process to ensure we have regular CPC concensus to move forward.
+   * Go/no go confirmation to the applicant that they are ready to have their application become public.
+   * Discussion/agreement between applicant, application manager and Foundation media team to agree on announce date.
 1. Project goes through process of adhering to [on-boarding checklist](#onboarding-checklist).
 1. Project presents to CPC for final acceptance via 2/3 supermajority vote and appropriate initial stage is determined.
 
+##### Initial fit checklist
+
+1. Is project open/willing to move to more open governance?
```

Comment:
As I've stated above in review, I take issue with the half-measure of "more open governance." I suggest we remove the word "more."

Reply:
Some applicants (specially those of small projects) might not now what "open governance" means. Maybe we should link to some examples?